.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -35,6 +35,20 @@ For more information, see the `CONTRIBUTING` guide.
 
 **Actual behavior:** [What actually happened]
 
-## Additional context
+## Additional Context
 
 Add any other context about the problem here..
+
+## Usage Information
+
+SDK Version - 
```

Comment:
Here we can some palce holder so that it will easy to replace. Like,
SDK Version: <SDK_VERSION_THAT_YOU_ARE_USING>

Reply:
Same goes for the node and the browser details